frontend/www/js/omegaup/components/common/GraderStatus.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <ul>
+    <hr class="dropdown-separator" />
+    <li class="grader grader-submissions">
+      <a class="grader-submissions-link" href="/arena/admin/">{{
+        T.wordsLatestSubmissions
+      }}</a>
+    </li>
+    <li class="grader grader-status">{{ graderStatusMessage }}</li>
+    <li class="grader grader-broadcaster-sockets" v-if="status === 'ok'">
+      Broadcaster sockets:
+      {{ graderInfo !== null ? graderInfo.broadcaster_sockets : '' }}
+    </li>
+    <li class="grader grader-broadcaster-sockets" v-else-if="error !== null">
+      API api/grader/status call failed:
+      <pre style="width: 40em;">{{ UI.escape(error) }}</pre>
```

Comment:
```suggestion
      <pre style="width: 40em;">{{ error }}</pre>
```

en Vue no es necesario hacer escaping :D

Reply:
Listo, se quita